common/src/main/java/io/pravega/common/concurrent/ExecutorServiceFactory.java

```diff
@@ -155,17 +149,18 @@ ScheduledExecutorService newScheduledThreadPool(int size, String poolName, int t
         ThreadFactory threadFactory = getThreadFactory(poolName, threadPriority);
 
         // Caller runs only occurs after shutdown, as queue size is unbounded.
-        ScheduledThreadPoolExecutor result = this.createScheduledExecutor.apply(size, threadFactory);
-
-        // Do not execute any periodic tasks after shutdown.
-        result.setContinueExistingPeriodicTasksAfterShutdownPolicy(false);
-
-        // Do not execute any delayed tasks after shutdown.
-        result.setExecuteExistingDelayedTasksAfterShutdownPolicy(false);
-
-        // Remove tasks from the executor once they are done executing. By default, even when canceled, these tasks are
-        // not removed; if this setting is not enabled we could end up with leaked (and obsolete) tasks.
-        result.setRemoveOnCancelPolicy(true);
+        ThreadPoolScheduledExecutorService result = this.createScheduledExecutor.apply(size, threadFactory);
+
+        //These aren't needed anymore as ThreadPoolScheduledExecutorService mandates all of them
```

Comment:
Let's remove the commented out code, but do leave a comment explaining that these options are mandated by default - that way, if we end up moving away from this executor, we know what we need to bring back without having surprises.